modules/addons/ratelimit/manifests/ratelimit-redis.yaml.tmpl

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ratelimit
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I still have this strong feeling we can use `tsb` image and just install it using `helm` (https://artifacthub.io/packages/helm/bitnami/redis)  it will be easier to keep up with the `deployment` changes instead of `having a tsb specific` way, practically if the customer is looking to deploy rate limiters they should be using their managed copy of `redis`... so by having an external redis as part of the repo...but matching the version 

technically we should not maintain own fork or "how to do redis", but having an external redis as part of this repo, will look more like an deployment pattern, in the same way as we do use external cert-manager. WDYT?

Reply:
Done